docs/get-started/sveltekit.md

```diff
@@ -0,0 +1,342 @@
+        },
+        navigating: {
+          route: {
+            id: '/storybook',
+          },
+        },
+        updated: true,
+      },
+    },
+  },
+};
+```
+
+The [available parameters](#parameters) are documented in the API section, below.
+
+### Mocking links
+
+The default link-handling behavior (e.g. when clicking an `<a href="..." />` element) is to log an action to the [Actions panel](../essentials/actions.md).
+
+You can override this by assigning an object to `parameters.sveltekit_experimental.hrefs`, where the keys are strings representing an href and the values define your mock. For example:
+
+```ts
+// MyComponent.stories.svelte
+export const MyStory = {
+  parameters: {
+    sveltekit_experimental: {
+      hrefs: {
+        '/basic-href': (to, event) => {
+          console.log(to, event);
+        },
+        '/root.*': {
+          callback: (to, event) => {
+            console.log(to, event);
+          },
+          asRegex: true,
+        },
+      },
+    },
+  },
+};
+```
+
+See the [API reference](#hrefs) for more information.
+
+## Writing native Svelte stories
+
+Storybook provides a Svelte addon maintained by the community, enabling you to write stories for your Svelte components using the template syntax. You'll need to take some additional steps to enable this feature.
+
+Run the following command to install the addon.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'svelte/svelte-csf-addon-install.yarn.js.mdx',
+   'svelte/svelte-csf-addon-install.npm.js.mdx',
+   'svelte/svelte-csf-addon-install.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info">
+
+The community actively maintains the Svelte CSF addon but still lacks some features currently available in the official Storybook Svelte framework support. For more information, see [the addon's documentation](https://github.com/storybookjs/addon-svelte-csf).
+
+</Callout>
+
+## API
+
+### Parameters
+
+This framework contributes the following [parameters](../writing-stories/parameters.md) to Storybook, under the `sveltekit_experimental` namespace:
+
+#### `forms`
+
+Type: `{ enhance: TK }`
+
+Provides mocks for the `$app/forms` module.
+
+##### `forms.enhance`
+
+Type: TK
+
+A callback that will be called when a form with `use:enhance` is submitted.
+
+#### `hrefs`
+
+Type: `Record<[path: string], (to: string, event: TK) => void | { callback: (to: string, event: TK) => void, asRegex?: boolean }>`
+
+If you have an `<a />` tag inside your code with the `href` attribute that matches one or more of the links defined (treated as regex based if the `asRegex` property is `true`) the corresponding `callback` will be called. If no matching `hrefs` are defined, an action will be logged to the [Actions panel](../essentials/actions.md). See [Mocking links](#mocking-links) for an example.
+
+#### `navigation`
+
+Type: TK
+
+Provides mocks for the `$app/navigation` module.
+
+##### `navigation.goto`
+
+Type: TK
```

Comment:
This would be the same as SvelteKit's own `goto` type - I don't know how to document that best? 

https://github.com/sveltejs/kit/blob/main/packages/kit/types/index.d.ts#L2055-L2068